platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -262,31 +266,57 @@ public void run() {
         });
     }
 
-    static MapController getInstance(GLSurfaceView view) {
-        return new MapController(view);
-    }
 
     /**
-     * Load a new scene file
+     * Load a new scene file.
+     * Use {@link #setSceneLoadListener(SceneLoadListener)} for notification when the new scene is
+     * ready.
      * @param path Location of the YAML scene file within the application assets
+     * @return Scene ID
      */
-    public void loadSceneFile(String path) {
-        loadSceneFile(path, null);
+    public int loadSceneFile(String path) {
+        return loadSceneFile(path, null);
     }
 
     /**
-     * Load a new scene file
+     * Load a new scene file asynchronously.
      * If scene updates triggers an error, they won't be applied.
+     * Use {@link #setSceneLoadListener(SceneLoadListener)} for notification when the new scene is
+     * ready.
      * @param path Location of the YAML scene file within the application assets
      * @param sceneUpdates List of {@code SceneUpdate}
+     * @return Scene ID
      */
-    public void loadSceneFile(String path, List<SceneUpdate> sceneUpdates) {
+    public int loadSceneFile(String path, List<SceneUpdate> sceneUpdates) {
```

Comment:
`loadSceneFileAsync` seems like a clearer name - at a minimum it will reduce confusion for people upgrading from older Tangram versions.